include/dali/core/util.h

```diff
@@ -344,6 +345,14 @@ static_assert(all_of<true, true, true>::value,
 static_assert(!all_of<true, false, true>::value,
               "Should return false_type when any of the values is false.");
 
+
+template <size_t N, typename T>
+std::array<T, N> uniform_array(const T& t) {
```

Comment:
How it differs from `ConstantFill` in tensor_test_utils.h? If not please reuse.

Reply:
ConstantFill is a testing utility. It takes a TensorView as a parameter and fills it based on it's shape.
This thing returns an std::array. I used it, because I wanted to have a way to express an uniform default value as a parameter to the filters.
I can't use span there, as it cannot be created from temporary. When I use const reference, the return value from this function is captured and it's lifetime is long enough to make it work.

Reply:
https://github.com/NVIDIA/DALI/blob/master/dali/test/tensor_test_utils.h#L341 this one works for iterable, you can create a std::array outside and call `ConstantFill` on it, or I have missed something.

Reply:
The point here is to do
```
void foo(const std::array<float, 5> &value = uniform_array<5>(1.0f));
```
So you have a default value of array parameter to function. Not really possible with ConstantFill.

Reply:
Understood. Still the question is if `include/dali/core/util.h`  is the place for the test utils.
@mzient ?

Reply:
This is not test utility.

Reply:
Right, after all we can use it to fill arguments array or something like this.